services/app/src/pages/game/Game.jsx

```diff
@@ -4,10 +4,23 @@ import ChessBoard from '../../components/chessBoard/ChessBoard';
 const Room = () => {
   const currentGameURL = window.location.href;
 
+  const copyGameURL = async () => {
+    try {
+      const url =
+        window.location.host + '/game/671e2367-86c3-453a-9df8-3c0048145b64';
+      await navigator.clipboard.writeText(url);
+      alert(`Link Copied To: \n ${url}`);
+    } catch (err) {
+      new Error(err);
+    }
+  };
+
   return (
     <div>
       <h1>GAME</h1>
-      <span id="game-url">{currentGameURL}</span>
+      <span id="game-url">
+        <a onClick={() => copyGameURL()}>{currentGameURL}</a>
```

Comment:
Please have a button (`<button/>`) next to the link. `<a/>` HTML tag is an anchor.